var/spack/repos/builtin/packages/libxkbcommon/package.py

```diff
@@ -18,8 +18,13 @@ class Libxkbcommon(AutotoolsPackage):
     version('0.8.2', sha256='7ab8c4b3403d89d01898066b72cb6069bddeb5af94905a65368f671a026ed58c')
     version('0.8.0', sha256='e829265db04e0aebfb0591b6dc3377b64599558167846c3f5ee5c5e53641fe6d')
 
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+    depends_on('m4', type='build')
```

Comment:
Why have these dependencies been added? Is there a missing `configure` somewhere?

Reply:
Good call.  I'm not sure why I added those.  Removed and build seems to happen just fine without them.  I also confirmed that the tarballs of both versions of libxkbcommon have `configure` scripts.

While making that change, I also rebase'd onto the latest upstream `develop`.